tests/YGMeasureTest.cpp

```diff
@@ -21,8 +21,24 @@ static YGSize _measure(YGNodeRef node,
   }
 
   return YGSize {
-      .width = 10,
-      .height = 10,
+      width = 10,
```

Comment:
it would be great if I could leave the ```.``` as this is the only way to let it compile on VS15. As it seems there is no need for it on clang, too, this shouldn't be a problem. What do you think?